# 緊急停止ボタンが押されると緊急停止モードに強制的に自動切り替えされるので、緊急停止ボタンを開いた時にこのメッセージを送って駆動モードに切り替えてください

bool ems_stop     # true → 緊急停止モードへ  ,  false → 駆動モードへ